### PR TITLE
Standardise mOTUs table merge

### PR DESCRIPTION
**Pull request by jfy133**
Standardise mOTUs merge to address #123 

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -99,19 +99,19 @@
                 },
                 "metaphlan3/metaphlan3": {
                     "branch": "master",
-                    "git_sha": "940d7fe9d63962e0e2ba0987e2893fb0aff832e3"
+                    "git_sha": "978087354eb72ac1f6e18a3f790fad9bc4d05840"
```
Comment (jfy133) [inline modules.json:102]:
```suggestion
                    "git_sha": "940d7fe9d63962e0e2ba0987e2893fb0aff832e3"
```

File: modules.json [hunk 2]
```diff
@@ -99,19 +99,19 @@
...
                 },
                 "minimap2/align": {
                     "branch": "master",
                     "git_sha": "1a5a9e7b4009dcf34e6867dd1a5a1d9a718b027b"
                 },
                 "minimap2/index": {
                     "branch": "master",
-                    "git_sha": "e745e167c1020928ef20ea1397b6b4d230681b4d"
+                    "git_sha": "73e062cf5e0ef346f65d73009b1b656299359fc5"
```
Comment (jfy133) [inline modules.json:110]:
```suggestion
                    "git_sha": "e745e167c1020928ef20ea1397b6b4d230681b4d"
```

**Review by sofstam — APPROVED** [review]
It looks good to me, but maybe would be good with an extra reviewer as well? :)

**Comment by jfy133** [comment]
It's exactly the same as the MetaPhlAn3 one, so if you're happy with that already it should be fine :)

**Comment by sofstam** [comment]
I do not have any comments/suggestions, more that I am not familiar with motus profiler itself 😃

**Pull request merged**